compose/src/main/kotlin/io/github/kakaocup/compose/intercept/interaction/ComposeInteraction.kt

```diff
@@ -3,16 +3,31 @@ package io.github.kakaocup.compose.intercept.interaction
 import androidx.compose.ui.test.SemanticsNodeInteraction
 import io.github.kakaocup.compose.intercept.operation.ComposeAction
 import io.github.kakaocup.compose.intercept.operation.ComposeAssertion
+import io.github.kakaocup.compose.node.builder.NodeProvider
 
 class ComposeInteraction(
-    private val semanticsNodeInteraction: SemanticsNodeInteraction
+    val nodeProvider: NodeProvider,
```

Comment:
`ComposeInteraction` is available in external libs through Interceptors. That's why it's better to give users more possibilities. The same approach is used in Kakao(Interceptors) and Kamulator.